index.js

```diff
@@ -10,7 +10,8 @@ var noop = function() {}
 module.exports = function(db) {
   var feed = {}
   var lock = mutexify()
- 
+  var valueEncoding = db.options.valueEncoding || 'binary'
```

Comment:
can valueEncoding passed in to options be different from db.options.valueEncoding? Seems like that would cause problems